File: marcos006/programacao_basica/Questao8.java
import java.util.Scanner;

    public class Questao8 {
        public static void main(String[] args) {
            Scanner s;
            s = new Scanner(System.in);

        System.out.println("Digite o lado do cubo:" );
        double l = s.nextDouble();
        double vc = l * l * l;
        System.out.println("O volume do cubo será de " + vc);

        System.out.println("Digite a altura do paralelepipedo: ");
        double a = s.nextDouble();
        System.out.println("Digite o comprimento de paralelepípedos: ");
        double c = s.nextDouble();
        System.out.println("Digite a base do paralelepípedo: ");
        double b = s.nextDouble();
        double vp = a * c * b;
        System.out.println("O volume do paralelepípedo será de " + vp);

        System.out.println("Digite o raio da esfera: ");
        double r = s.nextDouble();
        double ve = 4 * 3.14 * r * r * r/3;
        System.out.println("O volume da esfera será de " + ve);
    
    }
}

File: marcos006/programacao_basica/questao8.py
print(" ** Volume de um cubo ** ")
lado = float(input("Digite o lado do cubo: "))
volume_cubo = lado * lado * lado
print("O volume do cubo será de " + str(volume_cubo))

print(" ** Volume de um paralelepipedo  **")
altura = float(input("Digite a altura do paralelepipedo: "))
comprimento = float(input("Digite o comprimento do paralelepipedo: "))
base = float(input("Digite a base do paralelepipedo: "))
volume_paralelepipedo= altura * comprimento * base
print("O volume do paralelepipedo será de " + str(volume_paralelepipedo))

print(" ** Volume de uma esfera **")
raio = float(input("Digite o raio da esfera: "))
volume_esfera = 4 * 3.14 * raio * raio * raio/3
print("O volume da esfera será de " + str(volume_esfera))
